src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return new ColorSupplier().getRandomColor().toLowerCase();
+    }
```

Comment:
it is hardcoding
we should get color from the outside and don't hardcode color in Ball class
so let's create a constructor with two parameters - color and number 
```suggestion
    public String getColor() {
        return color;
    }
```